### PR TITLE
Fix issue with i3.metal pricing data not loading

### PR DESCRIPTION
**Pull request by meckstmd**
## Description
When Ariel retrieves pricing data from AWS it filters it out to only include the subset of data it needs.  One of the filters it uses is `Product Family` which it requires to be `Compute Instance`.  However some of the metal instance types have a Product Family of `"Compute Instance (bare metal)`.  I have opened a support case with AWS to determine what the reasoning is for this difference.

## Related Issue
https://github.com/yahoo/ariel/issues/3

## Motivation and Context
This change fixes an issue in which Ariel will not work if you have `i3.metal`, `r5.metal`, `r5d.metal` or `z1d.metal` instance types in any of your accounts.

## How Has This Been Tested?
This was tested by running it against a test account and verifying the cache file it produces includes the metal instance types listed above that it was missing before. After the `i3.metal` instance made it into our cost and usage report I then tested that it ran far enough to populate the `account_instance_summary` aurora DB table successfully.  I could not fully test the run because the Athena DB was in the `UPDATING` status.

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [X] I have read the **CONTRIBUTING** document.

## License  
I confirm that this contribution is made under the Apache License, Version 2.0 and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: ariel/get_ec2_pricing.py [hunk 1]
```diff
@@ -149,8 +149,27 @@ def load(config, locations=LOCATIONS):
         return prices
 
 def check_row(header_map, row):
-
-    if row[header_map['Product Family']] != 'Compute Instance':
+    
+    if row[header_map['Product Family']] != 'Compute Instance' and \
```
Comment (bastille) [inline ariel/get_ec2_pricing.py]: The pythonic way of doing this is `row[header_map['Product Family']] not in ['Compute Instance', 'Compute Instance (bare metal)`.  Just an FYI, it doesn't matter either way.

**Review by bastille — APPROVED** [review]
Looks good.  I had no idea Amazon exposed this as different products.

**Comment by meckstmd** [comment]
@bastille I just made the change to use the pythonic way.  I like it - it's much cleaner.  I also updated the comments in the code based on what I heard back from AWS about the different Product Families.

**Comment by meckstmd** [comment]
@bastille I have made some changes after discovering that it was using `i3.metal` as the family name which did not fit in the 5 characters allowed for the `instancefamily` in the `account_instance_summary` Aurora DB table.  Can you please look over these?

**Review by PeteSilva — APPROVED** [review]
👍

**Pull request merged**